tests/visualize/test_hsi_visualize.py

```diff
@@ -107,6 +107,10 @@ def test_visualize_hsi_with_hsi_object():
     # Check if the axes object is returned
     assert isinstance(ax, Axes)
 
+    # Check if title is set
```

Comment:
Add also testing lines for `test_attr_visualization` because you missed it with one title.